style.css

```diff
@@ -28,34 +28,100 @@
     cursor: pointer;
 }
 
+.fullHistory span:hover {
+    color: gray;
+    cursor: pointer;
+}
+
+.trashButton i:hover {
+    color: red;
+    opacity: .9;
+    cursor: pointer;
+}
+
 .calcField {
-    width: 25%;
+    width: 84%;
     margin: 0 auto;
-    border-style: 1px solid;
 }
 
-.topCalc {
-    margin: 20px 0;
+.opField {
+    width: 15%;
+    margin: 0 auto;
+}
+
+.topCalc{
+    justify-content: center;
+    display: flex;
+}
+
+.output {
+    width: 40%;
+    display: flex;
+}
+
+.history {
+    width: 40%;
```

Comment:
This goes with the comment below, since all of these items have a width of 40% you should consider making a single shared class that you put on all these elements for just the width. That way when you change the width in the media query you can just use a single class.